frontend/__mocks__/styleMock.js

```diff
@@ -0,0 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+module.exports = {};
```

Comment:
Could you add a small comment here indicating that it's to mock CSS imports required by Sprotty during the Jest tests as described here: https://jestjs.io/docs/webpack#handling-static-assets